src/dredd.coffee

```diff
@@ -41,6 +42,24 @@ class Dredd
     @transactions = []
     @runner = new Runner(@configuration)
     configureReporters(@configuration, @stats, @tests, @runner)
+    @logProxySettings()
+
+  logProxySettings: ->
+    proxySettings = []
+    for envVariable in Object.keys(process.env)
+      continue unless envVariable.toUpperCase() in PROXY_ENV_VARIABLES
```

Comment:
I could not use `in` in that case. I'd have to iterate over `PROXY_ENV_VARIABLES` and check each item by `caseless.has()`.